lms/static/js/edxnotes/views/search_box.js

```diff
@@ -32,6 +32,11 @@ define([
             this.logger.log('initialized');
         },
 
+        clearInput: function() {
+            // clear the search input box
+            this.$("#search-notes-input").val('');
```

Comment:
If we have `this.$el` then ideally we should use that.  `this.$el.find("#search-notes-input")`